source/usage-examples/updateMany.txt

```diff
@@ -7,14 +7,17 @@ Update Multiple Documents
 You can update multiple documents in a collection by using the ``UpdateMany()``
 method.
 
-The following example passes a query filter and an update parameter to the
-``UpdateMany()`` method, which matches documents in the
-``listingsAndReviews`` collection where the value of the ``address.market``
-field is "Sydney" and multiplies the ``price`` field in the matching
-documents by a factor of *1.15*:
+Example
+-------
 
 .. include:: /includes/usage-examples/run-example-tip.rst
 
+The following example performs the following on the
+``listingsAndReviews`` collection:
+
+- Matches documents in which the ``address.market`` is "Sydney"
```

Comment:
I wonder whether we should expect our usage example readers to know dot notation?
Maybe you could either:
- include a note about what dot notation is and link to more info (perhaps https://docs.mongodb.com/manual/core/document/#dot-notation)
- explain this is the notation they can use to search the market field of the address subdocument

Reply:
I'll update this to the second bullet